### DISCUSSION TITLE
include_init_images - ERROR

### DISCUSSION THREAD

**Discussion by Rogal80** [zero01101/openOutpaint#62] [Q&A] [answered]
Hi, when uploading the initial image, I always have this issue 
![obraz](https://user-images.githubusercontent.com/112553453/204084595-206152f9-223d-4f9e-9aab-ed053496ec55.png)
and notice also the same issue after generating first IMG from prompt and trying out paint other grids then bam this issue appear 
![obraz](https://user-images.githubusercontent.com/112553453/204085028-c97e7036-62d7-4b07-9de3-0c0381147713.png)

**Comment by seijihariki** [comment]
Hey, this is an issue with the webui, actually. It has some issues and fails to respond to img2img requests. The following is a pull request to resolve this issue: https://github.com/AUTOMATIC1111/stable-diffusion-webui/pull/4989.
You would probably be okay by fetching the patch-1 branch from https://github.com/moonshinegloss/stable-diffusion-webui.git as it is currently ahead of the base repo:
![image](https://user-images.githubusercontent.com/7552751/204089615-fcc64166-85fe-4281-a204-f44f7b936888.png)

Are you familiar with git?

**Comment by Rogal80** [comment]
Hi, I just know one or two commands, git pull, and git clone—what do I need to do to fix that?

**Comment by seijihariki** [comment, marked as answer]
In that case, to patch it temporarily it should be enough to 
```
$ git remote add moonshine https://github.com/moonshinegloss/stable-diffusion-webui
$ git fetch moonshine patch-1
$ git cherry-pick dae1fa7854a2aa6ac97fa92cda7ba63d3f0b584f
```

This may break future updates (git pull), so, until the pull request is accepted to the master branch, to update the webui you will need to do the following:
```
$ git reset --hard origin/master
$ git pull
$ git cherry-pick dae1fa7854a2aa6ac97fa92cda7ba63d3f0b584f
```

**Reply by seijihariki** [reply]
With the most recent updates to webui ([b2f17d](https://github.com/AUTOMATIC1111/stable-diffusion-webui/commit/b2f17dd367c5758e406dd22b78ad7456dac1957a) specifically), the issue should be gone.
